### ISSUE TITLE
The result is different between Python ver and CPP why? The merging Function?

### ISSUE THREAD

**Issue by David-dotcom666** [Gofinge/PyCannyLine#4]
![imgShowCPP](https://user-images.githubusercontent.com/84842493/161246016-b0761127-51b4-4274-898b-88dc3c8b1c5b.jpg)
![res](https://user-images.githubusercontent.com/84842493/161246292-994911bc-4ad8-44bd-b5d6-44a059d96b93.jpg)

**Comment by Gofinge** [comment]
Yes, I remembered differences exist between the two versions, but it has been almost one year, and I forgot the reason. The Python version was just for convenience to try some optimization. However, suppose you want to apply the original C++ version to your Python project. Then, I also provide a C++ version with Pybind, which you can directly try the original CannyLine in your python and much more fast.
